src/coreclr/jit/decomposelongs.cpp

```diff
@@ -2262,6 +2387,13 @@ void DecomposeLongs::TryPromoteLongVar(unsigned lclNum)
     {
         return;
     }
+#if defined(FEATURE_HW_INTRINSICS) && defined(TARGET_X86)
+    if (varDsc->lvIsParam)
+    {
+        // Promotion blocks combined read optimizations for SIMD loads of long params
+        return;
+    }
```

Comment:
In isolation, this change produced a small number of diffs and was mostly an improvement.  A few regressions show up in the SPMI reports, but the overall impact is good, especially considering the places we can load a long to vector with `movq`